### PR TITLE
Docker for all

### PR DESCRIPTION
**Pull request by machzqcq**

### CODE REVIEW

**Comment by machzqcq** [comment]
@dkhamsing - All I am trying to do is submit a link in README.md and the travis ci build is faling :( Confused ! Any help please ? Below is Travis error trace

`Issues :-(
> Links 
  1. [L044] 301 https://github.com/machzqcq/docker-for-all.git  → https://github.com/machzqcq/docker-for-all 
  2. [L441] 301 http://docs.gitlab.com/ce/container_registry/README.html  → https://docs.gitlab.com/ce/container_registry/README.html 
  3. [L610] 404 https://github.com/citadel/citadel  `

**Comment by dkhamsing** [comment]
Failure can be fixed separately 
Don't worry about it

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ about docker! (Tips: checkout her [dotfiles][jfrazelledotfiles] and her [dockerf
 * [Learn Docker](https://github.com/dwyl/learn-docker) Full environment set up, screenshots, step-by-step tutorial and more resources (video, articles, cheat sheets) by [@dwyl](https://github.com/dwyl)
 * [Docker Caveats](http://docker-saigon.github.io/post/Docker-Caveats/) What You Should Know About Running Docker In Production (written 11 APRIL 2016) __MUST SEE__
 * [How to Whale](https://howtowhale.com/) Learn Docker in your web browser, no setup or installation required.
+* [Docker for all - Developers, Testers, DevOps, Product Owners + Videos](https://github.com/machzqcq/docker-for-all.git) Docker Training Videos for all
```
Comment (dkhamsing) [inline README.md]: Can you remove .git

**Comment by dkhamsing** [comment]
Other issues can be fixed separately

**Comment by dkhamsing** [comment]
Thx

**Comment by pradeepinmar** [comment]
@dkhamsing Do I have to do anything else on my end, please let me know.

**Comment by machzqcq** [comment]
@dkhamsing Do I have to do anything else from my end , please let me know

**Comment by dkhamsing** [comment]
You're all set
Now @veggiemonk will evaluate your suggestion 

Thx

**Comment by vegasbrianc** [comment]
Hi @machzqcq thanks for the submission. Don't worry about the Build errors as we check all the links in the project to ensure everything is still alive and pointing to the correct URL's.

@dkhamsing thanks for assisting!

**Pull request merged**